src/natcap/invest/wave_energy.py

```diff
@@ -173,28 +167,31 @@
                     "type": "number",
                     "units": u.meter,
                     "about": (
-                        "user-defined wave height bins from smallest to "
+                        "User-defined wave height bins from smallest to "
                         "largest")
                 },
                 "[WAVE_PERIOD_BIN]": {
                     "about": (
                         "Captured wave energy for the given seastate "
                         "conditions described by the wave height (row) "
-                        "and wave period (column)"),
+                        "and wave period (column). Replace [WAVE_PERIOD_BIN] "
+                        "with the wave period bin in seconds, so that there "
+                        "is one column for each wave period bin, increasing "
+                        "from left to right."),
```

Comment:
It's been awhile since I've been in the Wave Energy model, but this additional sentence confused me. Are we trying to describe what the non index row/col values should be filled in with?

Reply:
I was trying to describe how all column headers after the first one are increasing wave period bin values. I agree this is confusing but I'm struggling to explain it well. I tried removing the whole `columns` spec and instead describing it in the `about` (2722ec3). I think that plus the examples in the UG make more sense for a matrix-style table. Let me know what you think.